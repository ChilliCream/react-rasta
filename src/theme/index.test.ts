import * as module from ".";

describe("index", () => {
  it("should verify the API shape", () => {
    expect(module).toMatchSnapshot();
  });
});
