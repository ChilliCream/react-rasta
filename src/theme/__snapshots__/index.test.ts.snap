// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`index should verify the API shape 1`] = `
Object {
  "Theme": Object {},
  "ThemeProperties": Object {},
  "gutterWidth": [Function],
  "styled": [Function],
}
`;
